lib/peridiod/config.ex

```diff
@@ -16,6 +16,9 @@ defmodule Peridiod.Config do
             cache_log_max_files: 0,
             cache_log_compress: true,
             cache_log_level: :debug,
+            distributions_download_before_install: false,
+            distributions_download_parallel_count: 0,
+            distributions_download_parallel_chunk_bytes: 1_048_576,
```

Comment:
Defaulted this to 1MB chunks. Dunno if that's a reasonable default or not, happy to adjust based on experience.

Reply:
lets do 5 MB ( not 5 MiB, careful )